dev-packages/e2e-tests/registrySetup.ts

```diff
@@ -85,8 +85,15 @@ export function registrySetup(): void {
       },
     );
 
-    if (publishImageContainerRunProcess.status !== 0) {
-      throw new Error('Publish Image Container failed.');
+    const statusCode = publishImageContainerRunProcess.status;
+
+    if (statusCode !== 0) {
+      if (statusCode === 137) {
+        throw new Error(
+          `Publish Image Container failed with exit code ${statusCode}, possibly due to memory issues. Consider increasing the memory limit for the container.`,
+        );
+      }
+      throw new Error(`Publish Image Container failed with exit code ${statusCode}`);
```

Comment:
I like the description, it's quite nice! 🙏🏼

For the sake of give my two cents: since `throw` won't will prevent the rest of the code to run, perhaps we may as well avoid the embedded if statements
```suggestion
    if (statusCode !== 0) {
      throw new Error(`Publish Image Container failed with exit code ${statusCode}`);
      }
    if (statusCode === 137) {
        throw new Error(
          `Publish Image Container failed with exit code ${statusCode}, possibly due to memory issues. Consider increasing the memory limit for the container.`,
        );
```

It's not so much for the performance benefits (since that's for testing), but I find it easier to read, as far as I'm concerned 😁 
Let me know if there is something I'm missing

Reply:
My bad, sorry, you're absolutely right, I don't know what I was thinking here. Ignore me, all good 😅